superbench/benchmarks/micro_benchmarks/dist_inference.py

```diff
@@ -390,7 +391,7 @@ def _process_data(self, step_times):
         Return:
             True if _process_data succeeds.
         """
-        if not self._process_numeric_result('step_times', step_times, cal_percentile=True):
+        if not self._process_numeric_result('step_times', step_times, reduce_type=ReduceType.MAX, cal_percentile=True):
```

Comment:
this is max on single gpu results? do you need `torch.dist.all_reduce(steptime, op=torch.dist.ReduceOp.MAX)` to get max result across gpus?

Reply:
Checked with @guoshzhao and @yukirora and the experiment results, it's max across ranks.